### PR TITLE
Completed Homework - Class 2, Week 1

### PR DESCRIPTION
**Pull request by JoshuaSLYoung**
Comfort Level: 
- A/C ===> 5/5
- Pyramid ===> 4/5

### CODE REVIEW

**Comment by JoshuaSLYoung** [comment]
Had some trouble figuring out the correct way to center my pyramid output in the command line

File: class02/josh/air_conditioning.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+if ((ac_functionality == "y") && (temperature > desired_temp))
+	puts "Turn on the A/C please!!!!!"
+elsif ((ac_functionality == "n") && (temperature > desired_temp))
+	puts "Fix the A/C now! It's hot!!!!!!"
+elsif ((ac_functionality == "n") && (temperature < desired_temp))
+	puts "Fix the A/C whenever you have the chance....it's cool!"
+else 
+	puts "Turn off the A/C - It's cold in here!"
+end
+
```
Comment (mobjohnson) [inline class02/josh/air_conditioning.rb]: Looks Good! Next time consider reducing the duplication of conditional statements using nested ifs

File: class02/josh/pyramid.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	while counter <= num_rows
+		puts (char_input * num_rows).center(30)
+		num_rows -= 1
+	end
+elsif (pyramid_direction =="n")
+	while counter <= num_rows 
+		puts (char_input * counter).center(30)
+		counter += 1
+	end
+else puts "You did not follow directions....."
```
Comment (mobjohnson) [inline class02/josh/pyramid.rb]: Nice input error handling!

File: class02/josh/pyramid.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# Ask the user if they want their pyramid upside-down and print it the way that they want it.
+puts "Do you want your pyramid upside-down? y or n?"
+pyramid_direction = gets.chomp
+
+
+counter = 1
+
+# Print out a simple ASCII art pyramid
+if (pyramid_direction == "y")
```
Comment (mobjohnson) [inline class02/josh/pyramid.rb]: You are missing a closing end for this if

**Pull request merged**